Our.Umbraco.TagHelpers/InlineSvgTagHelper.cs

```diff
@@ -101,11 +104,11 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
                     return;
                 }
 
-                fileContents = File.ReadAllText(file.PhysicalPath);
+                using var reader = new StreamReader(file.CreateReadStream());
```

Comment:
I have no problem with the changes, but more wanting to understand why the change to use a streamreader instead?

Reply:
`File.ReadAllText` can't be mocked due to being static, and I didn't want to create a set of dummy files. This makes it easier to test without having any affect on the functionality.